Poort8.Ishare.Service/Service/ServiceController.cs

```diff
@@ -133,9 +133,9 @@ public class ServiceController : ControllerBase
     {
         var authorization = Request.Headers.Authorization;
 
-        _logger.LogDebug("Received service PUT request with authorization header: {authorization}", authorization);
+        _logger.LogDebug("Received service PUT request with authorization header: {authorization}", authorization!);
 
-        var errorResponse = HandleAuthentication(authorization, out string? accessTokenAud);
+        var errorResponse = HandleAuthentication(authorization!, out string? accessTokenAud);
```

Comment:
Change authorization parameter in HandleAuthentication method to `string?`, since we do a nullability check in that method? Then the null forgiving parameters in this file aren't necessary.

Reply:
Fixed.